packages/cfd/src/Helpers/config.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { QRCode } from 'react-qrcode';
+import { TCFDsPlatformType } from 'Components/props.types';
+import { getDXTradeWebTerminalLink, platformsText, DERIVEZ_URL, platformsIcons, mobileDownloadLink } from './constants';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile, OSDetect, isDesktopOs } from '@deriv/shared';
+import { TCFDDashboardContainer } from 'Containers/props.types';
+
+export const getPlatformQRCode = (acc_type: TCFDsPlatformType) => {
+    const qr_code_mobile = isMobile() ? '100%' : '80%';
```

Comment:
change this variable name to the more descriptive one

```suggestion
    const qr_code_width = isMobile() ? '100%' : '80%';
```